heroku/deploy-a-django-app.md

```diff
@@ -28,24 +28,282 @@ preferred platform for hosting dynamic applications.
+
+The following instructions will help you deploy your properly configured application to the platform.
+
+### Install the Heroku CLI with the manifest plugin
+
+The fastest way to get a project up and running on Heroku is to use the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli). Before you start, make sure you have the CLI installed locally. Once you install the CLI, you'll need to [switch over to the CLI's beta version](https://devcenter.heroku.com/articles/build-docker-images-heroku-yml#creating-your-app-from-setup). This allows you to use the `manifest` CLI plugin, which you must install:
+
+```
+heroku update beta
+heroku plugins:install @heroku-cli/plugin-manifest
+```
+
+Confirm that you have the manifest plugin installed:
+
+```
+heroku plugins | grep manifest
+```
+
+If you don't see any output, [follow the official instructions for installing the
+plugin](https://devcenter.heroku.com/changelog-items/1441).
+
+### Create apps and pipelines for your project
+
+In order to deploy your project, you need to create Heroku apps for staging
+and production, and tie them together in a [pipeline](https://devcenter.heroku.com/articles/pipelines).
+To create these resources, start by defining the name of your app:
+
+```bash
+# This should be the same slug as your project's GitHub repo.
+export APP_NAME=<your-app-name-here>
+```
+
+Then, run the following Heroku CLI commands to create a staging app and a pipeline:
+
+```bash
+heroku create ${APP_NAME}-staging -t datamade --manifest
+heroku pipelines:create -t datamade ${APP_NAME} -a ${APP_NAME}-staging -s staging
+heroku pipelines:connect ${APP_NAME} -r datamade/${APP_NAME}
+```
+
+Your CLI output should look like this:
+```bash
+heroku create ${APP_NAME}-staging -t datamade --manifest 
+Reading heroku.yml manifest... done
+Creating ⬢ demo-app-staging... done, stack is container
+Adding heroku-postgresql... done
+https://demo-app-staging.herokuapp.com/ | https://git.heroku.com/demo-app-staging.git
+
+heroku pipelines:add ${APP_NAME}-staging -a ${APP_NAME}-staging -s staging
+Adding ⬢ demo-app-staging to datamade-app pipeline as staging... done
+```
+
+If you would like to set up a production app as well, run the following commands
+to create one and add it to your pipeline:
+
+```bash
+heroku create ${APP_NAME} -t datamade --manifest
+heroku pipelines:add ${APP_NAME} -a ${APP_NAME} -s production
+```
+
+Once you have the environments you need, enable [review
+apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+for your pipeline:
+
+```bash
+# Note that these need to be two separate commands due to an open Heroku bug,
+# since --autodeploy and --autodestroy require a PATCH request
+heroku reviewapps:enable -p ${APP_NAME}
+heroku reviewapps:enable -p ${APP_NAME} --autodeploy --autodestroy
+```
+
```

Comment:
Can we add a heading here? Something like `Set configuration variables for review apps and deployments`? Be sure to add this to the contents at the top, as well.

Reply:
Breaks that section up nicely: 9c03a68